exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
+let myName = "Negin";
+let greeting = "Hello, My name is ";
+let message = greeting + " " + myName;
```

Comment:
In here, there is no need to have the ` "  " ` to add extra space as you have already added it after the word `is  "`